keylime/tpm/tpm_main.py

```diff
@@ -29,8 +29,9 @@
 logger = keylime_logging.init_logging("tpm")
 
 
-class tpm(tpm_abstract.AbstractTPM):
+class tpm:
     tools_version: str = ""
+    EXIT_SUCCESS: int = 0
```

Comment:
As a constant, I would put it outside the class scope

Reply:
Done.